src/com/amazon/kinesis/streaming/agent/Agent.java

```diff
@@ -56,8 +57,7 @@ public static void main(String[] args) throws Exception {
         String logLevel = opts.getLogLevel() != null ? opts.getLogLevel() : (config != null ? config.logLevel() : null);
         int logMaxBackupFileIndex = (config != null ? config.logMaxBackupIndex() : -1);
         long logMaxFileSize = (config != null ? config.logMaxFileSize() : -1L);
-        Logging.initialize(logFile, logLevel, logMaxBackupFileIndex, logMaxFileSize);
-        final Logger logger = Logging.getLogger(Agent.class);
+        final Logger logger = LoggerFactory.getLogger(Agent.class);
```

Comment:
If we do this, it means that those other options are useless, right? The `logLevel, `logMaxFileSize`, etc. Shouldn't we remove them?